app/assets/javascripts/establishment_shot/charts.js

```diff
@@ -103,62 +139,55 @@ EstablishmentShot.Charts = (function () {
-                ]
-            }, smallChartDefaults()),
-            co2_sheet_transport_total_co2_emissions: $.extend({
-                left: true,
-                top: true,
-                fa_icon: 'f1b9',
-                series: [
-                    { key: 'co2_sheet_non_energy_emissions_transport' },
-                    { key: 'co2_sheet_transport_total_domestic_aviation_co2_emissions' },
-                    { key: 'co2_sheet_transport_total_domestic_freight_co2_emissions' },
-                    { key: 'co2_sheet_transport_total_private_transport_co2_emissions' },
-                    { key: 'co2_sheet_transport_total_public_transport_co2_emissions' }
-                ]
-            }, smallChartDefaults())
+            }
+        },
+        addNonEnergyAttributes = function() {
+            // Non energetic are first query/color (bottom of bar chart)
+            for ( var i = 0; i < queries.length; i++ ) {
+                queries[i] = [ nonEnergyQueries[i] ].concat(queries[i]);
+                smallColors[i] = [ nonEnergyColor ].concat(smallColors[i]);
+            }
+            total_chart_attributes.title += "_non_energy"
+            total_chart_attributes.series.forEach( function ( serie ) {
+                serie.key = serie.key.replace(/_only_energetic/g,"");
+            });
+        };
+
+    return {
+        setNonEnergy: function(n_e) { if(n_e) addNonEnergyAttributes() },
+        getCharts: function () {
+            count = 0;
+            return {
+                bar_chart: $.extend(totalChartDefaults(), total_chart_attributes),
+                co2_sheet_buildings_households_total_co2_emissions: $.extend({
+                    left: true,
+                    top: false,
+                    fa_icon: 'f015',
+                }, addQueries(), smallChartDefaults()),
+                co2_sheet_industry_energy_total_co2_emissions: $.extend({
+                    left: false,
+                    top: false,
+                    fa_icon: 'f275'
+                }, addQueries(), smallChartDefaults()),
+                co2_sheet_agriculture_total_co2_emissions: $.extend({
+                    left: false,
+                    top: true,
+                    fa_icon: 'f06c'
+                }, addQueries(), smallChartDefaults()),
+                co2_sheet_transport_total_co2_emissions: $.extend({
+                    left: true,
+                    top: true,
+                    fa_icon: 'f1b9'
+                }, addQueries(), smallChartDefaults())
+            };
         },
         getQueries: function () {
             var chart,
-                queries = [];
+                queries = [],
+                charts = this.getCharts();
 
-            for (chart in this.charts) {
-                this.charts[chart].series.forEach(function (serie) {
+            for (chart in charts) {
+                charts[chart].series.forEach(function (serie) {
```

Comment:
I think this can be ignored... it seems like a false-positive. I think it's warning about [this kind of error](https://stackoverflow.com/a/46027349), which doesn't apply in this case as far as I can tell. What do you think @sfp-kemp?